docs/conf.py

```diff
@@ -40,7 +40,7 @@
 # -- General configuration ---------------------------------------------------
 
 # require a recent version of Sphinx
-needs_sphinx = '4.4'
+needs_sphinx = '6.2'
```

Comment:
``tox -e docs`` breaks on my local setup:

```
    title = caption % part
            ~~~~~~~~^~~~~~
TypeError: not all arguments converted during string formatting
```

I am able to fix it by changing ``extlinks`` dictionary in ``docs/conf.py`` to

```
extlinks = {
    'issue': ('https://github.com/iqm-finland/qiskit-on-iqm/issues/%s', 'issue %s'),
    'mr': ('https://github.com/iqm-finland/qiskit-on-iqm/pull/%s', 'MR %s'),
}
```

Reply:
Fixed.